### PR TITLE
Add flot

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge.

File: recipes/flot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
```
Comment (isuruf) [inline recipes/flot/meta.yaml]: Can you please fix this as in the rest of the PRs?

Comment (sannykr) [reply recipes/flot/meta.yaml]: Thanks for the review. I have removed nodejs from run requirements, and added it to test/requirements

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Please let me know if you have more suggestions.

File: recipes/flot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  build:
+    - nodejs
+    
+test:
+  commands:
+    - npm list -g Flot
+  requires:
+    - nodejs
+   
+
```
Comment (scopatz) [inline recipes/flot/meta.yaml]: nitpick: one space between sections

File: recipes/flot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  home: https://github.com/flot/flot
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Attractive JavaScript charts for jQuery'
+  description: |
+    Flot is a Javascript plotting library for jQuery.  
+  doc_url: http://www.flotcharts.org/
+  dev_url: https://github.com/flot/flot
+extra:
```
Comment (scopatz) [inline recipes/flot/meta.yaml]: nitpick: one line between sections

**Comment by scopatz** [comment]
Thanks @sannykr! I think that this is basically ready

File: recipes/flot/meta.yaml (hunk 3 above)
Comment (sannykr) [reply recipes/flot/meta.yaml]: Thanks for the review. I have made the suggested changes

**Pull request merged**